store/postgres/src/entities.rs

```diff
@@ -700,7 +490,7 @@ impl Connection<'_> {
     ) -> Result<bool, Error> {
```

Comment:
The return type can be changed to `()` I believe.

Reply:
Likewise, that same commit will unceremoniously yank all that code.